types/node/v8/index.d.ts

```diff
@@ -4971,6 +4972,7 @@ declare module "tls" {
         SNICallback?: (servername: string, cb: (err: Error | null, ctx: SecureContext) => void) => void;
         sessionTimeout?: number;
         ticketKeys?: Buffer;
+        checkServerIdentity?: (host: string, cert: PeerCertificate) => Error | undefined;
```

Comment:
I think this should be in `ConnectionOptions` below not in `TlsOptions`.
`TlsOptions` is used in `tls.createServer()` which doesn't support `checkServerIdentity`.
`ConnectionOptions` is used in `tls.connect()` which supports `checkServerIdentity`.

Reply:
You are correct, sorry about that. This is also included already in `ConnectionOptions` so no further work is necessary. Feel free to close this.

Reply:
I have no rights to close issues from others. Please close it by yourself.